devel/qt4pas/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           qmake 1.0
+
+name                qt4pas
+version             2.5
+revision            0
+categories          devel pascal
+license             GPL-2
+maintainers         {@kamischi web.de:karl-michael.schindler} openmaintainer
+description         Pascal wrapper for Qt4
+
+long_description    The Free Pascal Qt4 binding allows Free Pascal to interface with the \
+                    C++ Library Qt. \
+                    \
+                    This binding does not cover the whole Qt4 framework but only the classes \
+                    needed by the Cross Platform Lazarus IDE to use Qt as a Widget set.
+
+homepage            https://web.archive.org/web/20220729012239/http://users.telenet.be/Jan.Van.hijfte/qtforfpc/fpcqt4.html
```

Comment:
An `archive.org` link? I feel like it might make more sense to mirror the code somewhere onto Github...

Reply:
finkproject has the nice feature to offer an archive, for hosting such source files. I'll file a corresponding feature request.

Reply:
@kamischi @herbygillot I have a Github setup here, feel free to borrow: https://github.com/macports/macports-ports/pull/19848